serverless/stacks/AppStack.ts

```diff
@@ -55,6 +55,32 @@ export function AppStack({ stack }: StackContext) {
         "packages/functions/src/datasets/user/all_of_type.handler",
       "GET /datasets/user/{type}/{filename}/columns":
         "packages/functions/src/datasets/user/columns.handler",
+      "DELETE /dataset/user/{type}/{filename}" :
+        "packages/functions/src/datasets/user/delete_userdata.handler",
+      "POST /trainspace": {
+        function: {
+          handler: "packages/functions/src/trainspace/create_trainspace.handler",
+          permissions: ["dynamodb"]
```

Comment:
what exact permissions does SST get from passing in "dynamodb"? Is principle of least privilege enforced here? @alantao912

Reply:
I just looked into this, there are more granular permissions such as dynamodb:PutItem, dynamodb:GetItem, etc. I will implement this.